docker/docker.go

```diff
@@ -225,15 +220,17 @@ func UseLocalOrPull(ctx context.Context, dClient *client.Client, hostname string
 }
 
 func RemoveContainer(ctx context.Context, dClient *client.Client, hostname string, containerName string) error {
-	err := dClient.ContainerRemove(ctx, containerName, types.ContainerRemoveOptions{})
+	err := dClient.ContainerRemove(ctx, containerName, types.ContainerRemoveOptions{Force: true})
```

Comment:
what does force option do? If the container failed to stop, will it remove this container? Any implications?

Reply:
the force option will remove the container even if its running, i followed the same way its implemented in rancher repo https://github.com/rancher/rancher/blob/master/pkg/rkeworker/docker.go#L137-L142